libraries/integration/Microsoft.Bot.Builder.Integration.AspNet.WebApi/BotMessageHandlerBase.cs

```diff
@@ -87,7 +87,7 @@ public BotMessageHandlerBase(BotFrameworkAdapter botFrameworkAdapter)
 
                         return bot.OnTurnAsync(context);
                     },
-                    cancellationToken);
+                    cancellationToken).ConfigureAwait(false);
```

Comment:
See earlier comment on not using `.ConfigureAwait(false)` in ASP.NET specific components.